packages/transaction_signing_gateway/lib/gateway/transaction_signing_gateway.dart

```diff
@@ -57,6 +57,12 @@ class TransactionSigningGateway {
         password: password,
       );
 
+  /// Updates the public details of the wallet using [PlainDataStore]
```

Comment:
no need to add the info regarding `PlainDataStore`. this is an implementation detail and its better to not give a hint about that to the library users

Reply:
Perfect. Updated this